### PR TITLE
How to create and run a C# unit testing project

### PR DESCRIPTION
**Pull request by Ushter21**
duplicate of #4188

### CODE REVIEW

**Comment by Ushter21** [comment]
Hello @zolomohan I have created a new PR. 
Duplicate of #4188

**Comment by hectorkambow** [comment]
@zolomohan lets wait until you get your new setup before we take on any new reviews. 👍

**Comment by zolomohan** [comment]
@hectorkambow I can take this. Working with a temporary setup, not an issue 😄

**Comment by Ushter21** [comment]
Hello, @zolomohan @hectorkambow I wish to enquire about the progress of my PR. Thank you.

**Comment by Ushter21** [comment]
Hello, @ahmadmardeni1 @hectorkambow @zolomohan . What's the progress of my PR. Thank you.

File: content/articles/how-to-create-and-run-a-.net-unit-testing -project/index.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
+### Introduction
+Any programming language must include some form of testing. We will use Visual Studio to test ASP.Net apps.
+Write a test code in Visual Studio. It's also used to run an ASP.Net application's test code. It becomes much easier to check for faults in an ASP.Net application this way. The testing module in Visual Studio provides the built-in capability. A test for an ASP.Net project is run right immediately.
```
Comment (zolomohan) [inline content/articles/how-to-create-and-run-a-.net-unit-testing -project/index.md:3]: The Introduction is not very strong. Please find a way to improve it.

**Review by zolomohan — CHANGES_REQUESTED** [review]
Hi @Ushter21 Please add proper spacing between lines and paragraphs.
Also, add proper indentation to code snippets.

**Comment by Ushter21** [comment]
Hello @zolomohan I have done the changes you requested. Thank you.

**Review by zolomohan — APPROVED** [review]

**Comment by hectorkambow** [comment]
Jumping on this one.

**Comment by hectorkambow** [comment]
👍 overall nice work - very detailed and straight forward tutorial 
 
Please review and suggestions/changes made by your peer reviewer and/or myself during a final review.

cc @zolomohan please be sure to go through the author index file as well to check for any mistakes and please do not forget to add the new meta image line 👍

**Pull request merged**